### ISSUE TITLE
PopOver needs 2 clicks to re-open if closed by yielded close action

### ISSUE THREAD

**Issue by roberkules** [kaliber5/ember-bootstrap#752]
After not being able to initially reproduce the issue on ember-twiddle and accidentally stumbling upon #750  I could reproduce the issue now, see the latest version of the [twiddle](https://ember-twiddle.com/e48db7c37c9055a016560bbc69dcca96?openFiles=templates.application.hbs%2C)

So the issue is, that if the popover is closed using the yielded close action, the next time it needs 2 clicks on the `CLICK ME` button to re-open the popover.

```hbs
<div id="ember-bootstrap-wormhole"></div>

<button class="btn btn-primary btn-lg">
  CLICK ME
  {{#bs-popover as |po|}}
    <button class="btn btn-lg btn-link" {{action po.close}}>Cancel</button>
  {{/bs-popover}}
</button>
```
---

Output from `ember bootstrap:info`:

```
Npm packages:
ember-bootstrap: ~2.4.0 -> 2.4.0
ember-cli: ~3.2.0 -> 3.2.0
bootstrap: ~4.1.3 -> 4.1.3
bootstrap-sass: n/a
ember-cli-sass: ~8.0.1 -> 8.0.1
ember-cli-less: n/a
Bower packages:
bootstrap: n/a
bootstrap-sass: n/a
ember-bootstrap configuration:
bootstrapVersion: 4
importBootstrapFont: false
importBootstrapCSS: false
```

**Comment by Techn1x** [comment]
I have the same problem as well... a temporary workaround could be to use the popovers' `visible` attribute but that is kind of difficult in my case

**Comment by Techn1x** [comment]
Although based on https://github.com/kaliber5/ember-bootstrap/issues/750 I tried setting `renderInPlace=true` and now it doesn't require 2 clicks

**Comment by simonihmig** [comment]
Thanks for the reproduction, and sorry for the late reply. Unfortunately I am a bit short on time. If someone could provide a PR with a failing test, that would certainly be helpful! Basically using the above example and doing the clicks in a test as describer in the twiddle...

**Comment by roberkules** [comment]
@simonihmig seems like this issue has been resolved, I upgraded to 2.7.x and it's working!
